### PR TITLE
Set default pg version to 12.1

### PR DESCRIPTION
**Pull request by tbalthazar**
This is a temporary change.
We'll set new default next week with #31.

### CODE REVIEW

**Comment by tbalthazar** [comment]
```
$ aws s3 ls s3://ci-database-binary/ --recursive | grep 2020-02-14 | grep 12.1
2020-02-14 16:26:27         32 .md5/cedar-14/postgresql-12.1.tgz.md5
2020-02-14 16:38:40         32 .md5/heroku-16-build/postgresql-12.1.tgz.md5
2020-02-14 16:31:54         32 .md5/heroku-16/postgresql-12.1.tgz.md5
2020-02-14 16:51:14         32 .md5/heroku-18-build/postgresql-12.1.tgz.md5
2020-02-14 16:44:48         32 .md5/heroku-18/postgresql-12.1.tgz.md5
2020-02-14 16:24:37   27354985 cedar-14/postgresql-12.1.tgz
2020-02-14 16:36:30   27865544 heroku-16-build/postgresql-12.1.tgz
2020-02-14 16:30:59   27864450 heroku-16/postgresql-12.1.tgz
2020-02-14 16:50:04   30406086 heroku-18-build/postgresql-12.1.tgz
2020-02-14 16:43:50   30403811 heroku-18/postgresql-12.1.tgz
```

File: README.md [hunk 1]
```diff
@@ -24,6 +24,4 @@ is experimental and subject to change.
 
 ## Releasing a new version
 
-Make sure you publish this buildpack in the buildpack registry
-
-`heroku buildpacks:publish heroku/ci-postgresql master`
+Follow the [playbook](https://salesforce.quip.com/exHCAbUbmN4U).
```
Comment (jabrown85) [inline README.md]: This should be updated to the merged destination of the https://github.com/heroku/engineering-docs/pull/1294

Comment (tbalthazar) [reply README.md]: @jabrown85 thanks, updated.

**Comment by tbalthazar** [comment]
Tested with no version specified:
```
-----> Postgresql app detected
       Using postgresql version: 12.1
       Using stack version: heroku-18
```

and with version 11 specified:
```
-----> Postgresql app detected
       Using postgresql version: 11.2
       Using stack version: heroku-18
```

**Pull request merged**